knowledge-base/replace-text-with-inline-element.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Replace existing text with Inline element
+description: Replace existing text with Inline element
+type: how-to
+page_title: Replace existing text with Inline element
+slug: replace-text-with-inline-element
+position: 0
+tags: text, inline, element, line, break, replace
+res_type: kb
+---
+
+<table>
+<thead>
+	<tr>
+		<th>Product Version</th>
+		<th>Product</th>
+		<th>Author</th>
+	</tr>
+</thead>
+<tbody>
+	<tr>
+		<td>2020.1.310</td>
+		<td>RadWordsProcessing</td>
+		<td><a href="https://www.telerik.com/blogs/author/martin-velikov">Martin Velikov</a></td>
+	</tr>
+</tbody>
+</table>
+
```

Comment:
The other KBs have a different structure here.

Reply:
Both are valid and produce the same result.